x/staking/types/msg.go

```diff
@@ -71,8 +71,8 @@ func (msg MsgCreateValidator) GetSignBytes() []byte {
 	return sdk.MustSortJSON(bz)
 }
 
-// ValidateBasic implements the sdk.Msg interface.
-func (msg MsgCreateValidator) ValidateBasic() error {
+// Validate validates the MsgCreateValidator sdk msg.
+func (msg MsgCreateValidator) Validate() error {
 	// note that unmarshaling from bech32 ensures both non-empty and valid
```

Comment:
why does this stay?

Reply:
https://github.com/cosmos/cosmos-sdk/pull/15820#pullrequestreview-1383697017

Because we need to verify this 3 times.